src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 1;
+
+  const result = (a) => {
+    return (b) => {
+      if (count <= 3 || count % 2 !== 0) {
+        count++;
+
+        return a + b;
+      } else {
+        count++;
+
+        return 'Bzzz... Error!';
+      }
+    };
```

Comment:
Can you write code in `else` body outside `else`?
```suggestion
      }
     
      count++;

      return 'Bzzz... Error!';
    };
```